dracut/endless.conf

```diff
@@ -9,3 +9,7 @@ early_microcode="yes"
 # between nouveau and nvidia.
 # (nouveau cannot be unloaded after it has bound to a device)
 omit_drivers="nouveau"
+
+# Ship the bfq IO scheduler module in the initrd, as it's relied upon by
+# our udev rules that detect block devices
+add_drivers+="bfq"
```

Comment:
Does this also cause the module to be loaded? Or is that caused by the udev rule? In my testing, adding `scsi_mod.use_mq_default=1` did not cause `bfq` to be loaded and (IIRC) nor did echoing `bfq` into `/sys/block/sda/queue/scheduler` (but I could be wrong).

Reply:
I don't believe this causes the module to be loaded, just to be included in the initrd. echoing bfq into the queue/scheduler file did that for me. I believe this is achieved by a $foo-iosched alias which is provided by the bfq module. Some other modules are literally named $foo-iosched.

Reply:
Huh – apparently I did not recall correctly. I've tested locally that the module is loaded I write to `queue/scheduler`.